### PR TITLE
Fix CAPI getting-started  tutorial

### PR DESCRIPTION
**Pull request by maci3jka**
With the current state of docs with the following steps, the user cannot deploy Canonical K8s using CAPI this pull request fixes that issue.

fixes: KU-1468, #625

### CODE REVIEW

File: docs/src/capi/tutorial/getting-started.md [hunk 1]
```diff
@@ -183,7 +191,7 @@ Set the respective environment variables by editing the rc file as needed
 before sourcing it. Then generate the cluster manifest:
 
 ```
-source ./templates/<infrastructure-provider>/template-variables.rc
+source ./templates/<infrastructure-provider>/cluster-template.rc
```
Comment (maci3jka) [inline docs/src/capi/tutorial/getting-started.md:194]:
```suggestion
source ./templates/<infrastructure-provider>/template-variables.rc
```

File: docs/src/capi/tutorial/getting-started.md [hunk 2]
```diff
@@ -18,13 +18,21 @@ sudo install -o root -g root -m 0755 clusterctl /usr/local/bin/clusterctl
...
 ### Configure clusterctl
 
-`clusterctl` contains a list of default providers. Right now, {{product}} is
-not part of that list. To make `clusterctl` aware of the new 
-providers, we need to add a clusterctl configuration file.
+`clusterctl` contains a list of default providers. Right now, {{product}} is 
+not yet part of that list. To make `clusterctl` aware of the new
+providers, we need to add canonical providers to configuration
+file. Edit `~/.cluster-api/clusterctl.yaml` and add the following:
+
```
Comment (maci3jka) [inline docs/src/capi/tutorial/getting-started.md:25]:
```suggestion
```

**Comment by maci3jka** [comment]
merge after https://github.com/canonical/cluster-api-k8s/pull/49

**Review by bschimke95 — APPROVED** [review]
LGTM

File: docs/src/capi/tutorial/getting-started.md [hunk 3]
```diff
@@ -18,13 +18,20 @@ sudo install -o root -g root -m 0755 clusterctl /usr/local/bin/clusterctl
...
-`clusterctl` contains a list of default providers. Right now, {{product}} is
-not part of that list. To make `clusterctl` aware of the new 
-providers, we need to add a clusterctl configuration file.
-
-```
-mkdir -p ~/.config/cluster-api
-curl -L https://raw.githubusercontent.com/canonical/cluster-api-k8s/main/clusterctl.yaml -o ~/.config/cluster-api/clusterctl.yaml
+`clusterctl` contains a list of default providers. Right now, {{product}} is 
+not yet part of that list. To make `clusterctl` aware of the new
+providers, we need to add canonical providers to configuration
```
Comment (evilnick) [inline docs/src/capi/tutorial/getting-started.md:23]:
```suggestion
providers we need to add add them to the configuration
```

**Review by evilnick — APPROVED** [review]
Just a word tweak. Thanks! :cake:

**Pull request merged**